staff/sergio-arce/spotify-template/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Spotify</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main>
+        <!-- header -->
+        <header class="header">
+            <img class="header__logo" src="./images/logo-spotify.png" alt="">
+            <button class="header__button">VIEW ON SPOTIFY</button>
+        </header>
+        <!-- hader -->
+  
+        <!-- star -->
+        <section class="start">
+            <h2 class="start__">¡Que empieza el fin de semana!</h2>
```

Comment:
@sergi23 this is not a valid class with BEM. Maybe you meant `start__title`?

Reply:
donde